app/models/validations/property_validations.rb

```diff
@@ -57,4 +57,27 @@ def validate_uprn(record)
 
     record.errors.add :uprn, I18n.t("validations.property.uprn.invalid")
   end
+
+  def validate_la_in_england(record)
+    return unless record.form.start_year_2025_or_later?
+
+    if record.is_general_needs?
+      return unless record.la
+      return if record.la.in?(LocalAuthority.england.pluck(:code))
+
+      record.errors.add :la, I18n.t("validations.lettings.property.la.not_in_england")
+      record.errors.add :postcode_full, I18n.t("validations.lettings.property.postcode_full.not_in_england")
+      record.errors.add :uprn, I18n.t("validations.lettings.property.uprn.not_in_england")
+      record.errors.add :uprn_confirmation, I18n.t("validations.lettings.property.uprn_confirmation.not_in_england")
+      record.errors.add :uprn_selection, I18n.t("validations.lettings.property.uprn_selection.not_in_england")
+      record.errors.add :startdate, I18n.t("validations.lettings.property.startdate.not_in_england")
+    elsif record.is_supported_housing?
+      return unless record.location
+      return if record.location.location_code.in?(LocalAuthority.england.pluck(:code))
```

Comment:
Do we (or should we) validate this when locations are created?

Reply:
I thought that, but it might be a bit difficult if the locations are used across different years